src/System.Runtime.Handles/src/System.Runtime.Handles.csproj

```diff
@@ -18,10 +19,8 @@
     <Compile Include="System\IO\HandleInheritability.cs" />
     <Compile Include="System\Threading\WaitHandleExtensions.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcore50aot'">
-    <TargetingPackReference Include="System.Private.CoreLib" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
     <TargetingPackReference Include="System.Private.Interop" />
```

Comment:
I do not think it make sense to have mix of forwards to System.Runtime.* and System.Private.*. It should be either all System.Runtime.* or all System.Private.*.